zebra-rpc/src/methods.rs

```diff
@@ -482,23 +727,33 @@ where
     State::Future: Send,
     Tip: ChainTip + Clone + Send + Sync + 'static,
     AddressBook: AddressBookPeers + Clone + Send + Sync + 'static,
+    BlockVerifierRouter: Service<zebra_consensus::Request, Response = block::Hash, Error = zebra_consensus::BoxError>
+        + Clone
+        + Send
+        + Sync
+        + 'static,
+    <BlockVerifierRouter as Service<zebra_consensus::Request>>::Future: Send,
+    SyncStatus: ChainSyncStatus + Clone + Send + Sync + 'static,
 {
     /// Create a new instance of the RPC handler.
     //
     // TODO:
     // - put some of the configs or services in their own struct?
     #[allow(clippy::too_many_arguments)]
     pub fn new<VersionString, UserAgentString>(
+        net: Network,
+        mining_conf: config::mining::Config,
```

Comment:
Nitpick/Optional: It would be nice to use the full terms here and to keep `chain_tip` named `latest_chain_tip` for discoverability, but the field names are still what's expected so it's not too significant either way.

```suggestion
        network: Network,
        mining_config: config::mining::Config,
```

Reply:
Addressed in [Rename vars in RpcImpl::new](https://github.com/ZcashFoundation/zebra/pull/9459/commits/1b810098c78f57bbf57565f9090b074264c8ab87).